src/frontend/dpia-webapp/src/assets/main.css

```diff
@@ -0,0 +1,35 @@
+@import "./base.css";
```

Comment:
Should we use Sass for this ?

Reply:
this is how vue init's a project, this import wasn't generated by me. we can look into changing this in the future if we require.